howso/direct/core.py

```diff
@@ -3253,12 +3083,14 @@ def _set_label(self, label: str, payload: Any) -> None:
         self.amlg.set_json_to_label(
             self.handle, label, json.dumps(payload))
```

Comment:
`_get_label` and `_set_label` still reference the now removed `self.handle`

Reply:
these methods just don't make sense anymore... gonna remove them as well.